src/Main.js

```diff
@@ -2,30 +2,74 @@ import { Component } from 'react';
 import HornedBeasts from './HornedBeasts.js';
 import Row from "react-bootstrap/Row"
 import Container from 'react-bootstrap/Container';
+import  Form  from 'react-bootstrap/Form';
 
 
+
+const numberOfHorns = [1, 2, 3, 100];
+
 class Main extends Component{
+
+  constructor(props){
+    super(props);
+    this.state ={
+      horns : numberOfHorns
+    }
+  }
+
+  handleChange = (event) => {
+    const selection = event.target.value;
+    console.log(selection);
+    let selectedHornArr;
+
+    if (selection === '1'){
+      selectedHornArr = numberOfHorns.filter(horn => (horn === 1))
+    } else if (selection === '2'){
+      selectedHornArr = numberOfHorns.filter(horn => (horn ===2))
+    }else if (selection === '3'){
+      selectedHornArr = numberOfHorns.filter(horn => (horn ===3)) 
+    }else if (selection === '100'){
+      selectedHornArr = numberOfHorns.filter(horn => (horn ===100))
+    }else if (selection === 'all'){
+      selectedHornArr = numberOfHorns
+    }
+
+    this.setState({horns: selectedHornArr})
```

Comment:
Good work here too!

Another way to do this would be to use one filter(). 
We could import json data  and filter over data. If beast.horns === selection, then set the state to be whatever was filtered.  

Just some food for thought 👍